test/support/models/article.ex

```diff
@@ -3,6 +3,8 @@ defmodule ExMachina.Article do
 
   schema "articles" do
     field :title, :string
+
     belongs_to :author, ExMachina.User
+    belongs_to :editor, ExMachina.User
```

Comment:
Thoughts on this, instead? https://github.com/thoughtbot/ex_machina/pull/124/commits/9a0cc9dbeafe4d7a19d080d01dccf60ad2003572